uiowa/uiowa
# Updated Classroom content type with basic fields

Related issue #5769 

# How to test
Setup one-liner:
```
ddev blt ds --site classrooms.uiowa.edu && ddev drush @classrooms.local uli
```
- `ddev blt ds --site classrooms.uiowa.edu`
- `ddev drush @classrooms.local uli`

Test that the content type meets the criteria described in https://wiki.uiowa.edu/pages/viewpage.action?spaceKey=drupaldev&title=Content+type+creation+notes

Test that these fields work as expected:

- Featured Image
- Image gallery
- Seating Chart 
- Video
- Map 

Double check that the fields here match the fields described in the "Fields" section on https://github.com/uiowa/uiowa/issues/5769.
